demisto_sdk/commands/common/constants.py

```diff
@@ -197,6 +197,8 @@ class FileType(str, Enum):
     FileType.XSIAM_REPORT: 'XSIAM Reports',
     FileType.TRIGGER: 'Triggers Recommendations',  # https://github.com/demisto/etc/issues/48153#issuecomment-1111988526
     FileType.WIZARD: 'Wizards',
+    FileType.XDRC_TEMPLATE: 'XDRC Templates',
+    FileType.XDRC_TEMPLATE_YML: 'XDRC Templates',
```

Comment:
why do we need to differentiate between the two? what's the difference between this content type and integration type that has multiple files?